charts/openstack-cluster/values.yaml

```diff
@@ -93,6 +93,12 @@ registryMirrors:
   registry.k8s.io:
     - https://quay.io/v2/azimuth/registry.k8s.io
 
+# Proxy configuration for containerd.
+proxy:
```

Comment:
I think `proxy` is too broad - it implies that we are setting the proxies for all components in the cluster. Maybe `containerdProxy` instead?